include/util/static_rtree.hpp

```diff
@@ -65,11 +79,20 @@ class StaticRTree
     using EdgeData = EdgeDataT;
     using CoordinateList = Vector<util::Coordinate>;
 
+    static constexpr auto UNALIGNED_LEAF_PAGE_SIZE =
+        BRANCHING_FACTOR * sizeof(EdgeDataT) + sizeof(Rectangle) + sizeof(std::uint32_t);
```

Comment:
This yields a page size of `516` bytes, which rounds out to 1024. Do you need to use `BRANCHING_FACTOR` for the leaves too?